raiden/smart_contracts/NettingChannelLibrary.sol

```diff
@@ -56,6 +56,19 @@ library NettingChannelLibrary {
         _;
     }
 
+    modifier inNonceRange(Data storage self, bytes message) {
+        uint64 nonce;
+        nonce = getNonce(message);
+        if (nonce < self.opened * (2**32) || nonce > (self.opened + 1) * (2**32))
```

Comment:
We have an off-by-one here, it should be:
```
if (nonce < self.opened * (2**32) || nonce >= (self.opened + 1) * (2**32))
```
since `(self.opened + 1) * (2**32)` is a valid nonce of the *next* range.